packages/pymedphys_dicom/src/pymedphys_dicom/dicom/identifying_keywords.json

```diff
@@ -0,0 +1,250 @@
+[
+  "AccessionNumber",
+  "AcquisitionComments",
+  "AcquisitionContextSequence",
```

Comment:
Thanks Matt :). At some point it would be great to link to this list in the docs, and maybe also link to DICOM Supplement 142, and say this list was generated from that supplement.

Reply:
Yep, good point!